01-IAM/README.md

```diff
@@ -1,64 +1,70 @@
 
 # Cloud Foundation Toolkit Lab - 01-IAM
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2Fterraform-google-modules%2Fcloud-foundation-training&cloudshell_open_in_editor=main.tf&cloudshell_tutorial=README.md&cloudshell_working_dir=01-IAM)
 
 ## Prerequisite
 
-Complete previous labs
-* **[00-Setup](../00-Setup/README.md)**
+Complete previous labs:
```

Comment:
Should we add `<walkthrough-project-setup></walkthrough-project-setup>` in the 00-Setup instruction to guide project selection?

Reply:
That's a great idea. Let's consider that in the `getting-started` branch?